modules/core/assets/ships/imperialCapital/imperialCapital.json

```diff
@@ -0,0 +1,443 @@
+{
+    "size": 3.5,
+    "maxLife": 800,
+    "type": "big",
+    "price": 20000,
+    "e1Pos": "0.08 0.37",
+    "e2Pos": "0.08 0.63",
+    "displayName": "Imperial Capital Ship",
```

Comment:
Alternatively we could give it a flavor name and start referring to a _class_ of capital ships :-)

That's probably a good idea in either case. Gives us an easy way to associate all capital ships with class behavior, like making them avoid dipping below the orbital band of a planet under normal circumstances.